### DISCUSSION TITLE
Possible to add global keybindings?

### DISCUSSION THREAD

**Discussion by chaseconey** [filamentphp/filament#14286] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.113

### How can we help you?

I am curious if it's possible to set a global keybinding for the default actions that are on most resources (like Edit).

It would be nice to not have to go to every resource and configure "E" to trigger the "Edit" action, for example.

Is this possible currently?

**Comment by leandrocfe** [comment, marked as answer]
Add in your AppServiceProvider

```php
public function boot(): void
{
    EditAction::configureUsing(function (EditAction $action) {
        $action->keyBindings('E');
    });
}
```

**Reply by chaseconey** [reply]
Amazing. Thanks @leandrocfe!
